tests/unit/keyspace.tcl

```diff
@@ -193,6 +193,22 @@ start_server {tags {"keyspace"}} {
         }
     } 
 
+    test {COPY basic usage for string encoded shared integer number} {
+        r set intkey{t} 1
+        assert {[r object refcount intkey{t}] == 2147483647}
+        r copy intkey{t} newintkey1{t}
+        assert {[r object refcount newintkey1{t}] == 2147483647}
+        set old [lindex [split [r debug object intkey{t}]] 1]
+        set new1 [lindex [split [r debug object newintkey1{t}]] 1]
+        assert {$old eq $new1}
+        r config set maxmemory 10gb
+        r config set maxmemory-policy allkeys-lru
```

Comment:
Two of the tests failed because this test needs to be tagged with "{needs:config-maxmemory}"